save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -5,137 +5,407 @@
+    }
+
+    private fun submitRequest(url: String, headers: Headers, body: dynamic) {
         GlobalScope.launch {
-            responseFromExecutionRequest = post("${window.location.origin}/submitExecutionRequest", headers, jsonExecution)
+            responseFromExecutionRequest = post(window.location.origin + url, headers, body)
         }.invokeOnCompletion {
             if (responseFromExecutionRequest.ok) {
                 window.location.href = "${window.location.origin}/history"
             } else {
                 setState {
                     isErrorOpen = true
-                    errorText = responseFromExecutionRequest.statusText
+                    errorLabel = "Error from backend"
+                    errorMessage = responseFromExecutionRequest.statusText
                 }
             }
         }
     }
 
-    @Suppress("TOO_LONG_FUNCTION")
+    @Suppress("TOO_LONG_FUNCTION", "LongMethod", "ComplexMethod")
     override fun RBuilder.render() {
         // modal window for configuring tests run - initially hidden
-        runErrorModal()
+        runErrorModel()
+        runLoadingModal()
+        numberOpenningCard = 1
         // Page Heading
         div("d-sm-flex align-items-center justify-content-between mb-4") {
             h1("h3 mb-0 text-gray-800") {
-                +"Project ${props.executionRequest.project.name}"
+                +"Project ${project.name}"
             }
         }
 
         div("row") {
             child(cardComponent {
                 div("text-center") {
-                    div("pb-4") {
-                        h6(classes = "d-inline") {
-                            +"Path to property file: "
+                    attrs.id = "accordion"
+                    div("card shadow mb-4") {
+                        div("card-header") {
+                            attrs.id = "headingFirst"
+                            button(classes = "btn btn-link collapsed") {
+                                attrs["data-toggle"] = "collapse"
+                                attrs["data-target"] = "#collapseFirst"
+                                attrs["aria-expanded"] = "true"
+                                attrs["aria-controls"] = "collapseFirst"
+                                h6("m-0 font-weight-bold text-primary") {
+                                    +"Upload project as git url"
+                                }
+                                attrs.onClickFunction = {
+                                    if (numberOpenningCard == 1) {
+                                        numberOpenningCard = 0
+                                    } else {
+                                        numberOpenningCard = 1
+                                    }
+                                }
+                            }
+                        }
+                        // Collapse card to load git url
+                        div("collapse show") {
```

Comment:
Can we place `show` class depending on `numberOpenningCard`?

Reply:
Did it